openvino_tensorflow/ie_backend_engine.cc

```diff
@@ -26,6 +26,18 @@ IE_Backend_Engine::~IE_Backend_Engine() {}
 void IE_Backend_Engine::load_network() {
   if (m_network_ready) return;
 
+  if (std::getenv("OPENVINO_TF_ENABLE_PERF_COUNT"))
```

Comment:
I think we can read the environment variables once in BackendManager and set a variable there. Then, we can call something like BackendManager::PerfCountEnabled(). So, we don't need to read the environment variable for each cluster.